std_msgs/Header header
sensor_msgs/NavSatFix position
geometry_msgs/Quaternion orientation